src/api/app/services/user_daily_contribution.rb

```diff
@@ -0,0 +1,45 @@
+class UserDailyContribution
+  attr_accessor :user, :date
+
+  def initialize(user, date)
+    @user = user
+    @date = date
+  end
+
+  def call
+    { comments: number_of_comments_for_date,
+      requests_reviewed: number_of_reviews_done_per_day,
+      commits: number_of_commits_done_per_day,
+      requests_created: requests_created_for_date }
+  end
+
+  private
+
+  def requests_created_for_date
+    user.requests_created.where('date(created_at) = ?', date).pluck(:number)
+  end
+
+  def number_of_comments_for_date
+    user.comments.where('date(created_at) = ?', date).count
+  end
+
+  def number_of_reviews_done_per_day
+    ReviewsFinder.new.completed_by_reviewer(user)
+                 .where('date(reviews.created_at) = ?', date)
```

Comment:
I'm not sure if we need to cast the date here, but would be nice if we are consistent. So either we cast `date` everywhere or we don't. Bonus would be a spec for that.

Reply:
I've added the casts in all places where needed.